src/presets/aws-lambda-edge.ts

```diff
@@ -0,0 +1,167 @@
+import { resolve } from "pathe";
+import { defineNitroPreset } from "../preset";
+import { Nitro } from "../types";
+import { writeFile } from "../utils";
+
+export const awsLambdaEdge = defineNitroPreset({
+  entry: "#internal/nitro/entries/aws-lambda-edge",
+  externals: true,
```

Comment:
this can't be a boolean

Reply:
It was unnecessary property so remove it.